public/app/features/org/SelectOrgPage.tsx

```diff
@@ -18,29 +18,33 @@ const navModel = {
   },
 };
 
-const getUserOrgs = async () => {
-  return await getBackendSrv().get('/api/user/orgs');
-};
+function mapStateToProps(state: StoreState) {
+  return {
+    userOrgs: state.user.orgs,
+  };
+}
 
 const mapDispatchToProps = {
   setUserOrganization,
+  getUserOrganizations,
 };
 
-const connector = connect(null, mapDispatchToProps);
+const connector = connect(mapStateToProps, mapDispatchToProps);
 
 type Props = ConnectedProps<typeof connector>;
 
-export const SelectOrgPage: FC<Props> = ({ setUserOrganization }) => {
-  const [orgs, setOrgs] = useState<UserOrg[]>();
+export const SelectOrgPage: FC<Props> = ({ setUserOrganization, getUserOrganizations, userOrgs }) => {
+  const [orgs, setOrgs] = useState<UserOrg[]>(userOrgs);
```

Comment:
we don't need this anymore. now that `userOrgs` is set in redux, we can remove this state entirely and replace the use of `orgs` with `userOrgs` further down in the component